### DISCUSSION TITLE
Autoinbetween in xsheet/timeline

### DISCUSSION THREAD

**Discussion by adamearle** [opentoonz/opentoonz#6793] [Roadmap]
**The Problem?**
The issue is the user has to perform multiple tasks to achieve a tweening result. 

1. A user inserts a keyframe between two drawings (Blank frames).
2. User selects the frames that are needed to be tweened in the "Level Strip Window".
3. User clicks "Inbetween button" in the "level strip window" for the drawings to update.
4. User, Alt-click, and drags the frames from the level strip editor to the Xsheet/ timeline to update the frames.

**Solution**
Macro-like behavior to combine some of OpenToonz's basic functionality together to make a one-button process creating a better user experience.
A user can normally stretch out a frame hold the frame as long as they want. The Proposed solution is the same idea except the user would hold down the "alt key" while dragging the frame. the behavior would a 

1. fill the typical "hold area" with blank keyframes
2. Then automatic in-between.

This would create the illusion of automated in-between behavior still giving OpenToonz that manual touch.

**Comment by artisteacher** [comment]
Faster in-betweening could only be beneficial!

**Comment by RodneyBaker** [comment]
Probably related to #1423 which at least at one time had a bounty placed upon it.

**Comment by RodneyBaker** [comment]
Tagging this with roadmap as Tahoma2D has added auto inbetween to the timeline.
Not quite the request here but it might suffice as a near workflow.
